input/fsh/treatment-consent-profile.fsh

```diff
@@ -12,7 +12,7 @@ A profile of the Consent resource for modeling treatment consents
 * date 1..1
 * period 1..1
 * grantor 1..1
-* grantee 1..1
+* grantee 1..
```

Comment:
why would we want to restrict this for only one? I think there are legitimate use cases where the permission is granted to more than one individual or a team or a role.

Reply:
Yes I agree. Sorry that was my intention. 1..* not 1..

Reply:
What is the syntax for 1 or more?

Reply:
this is actually correct syntax for 1..* sorry I mixed up the left and right panel here :D

Reply:
No issues! Yes I was fixing the fact that it was only 1 before.